src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -728,22 +717,13 @@ void AutomationInstrumentClipView::renderAutomationEditor(ModelStackWithTimeline
 
 	if (modelStackWithParam && modelStackWithParam->autoParam) {
 
```

Comment:
cleaned up code. refactored code for getting effective length out to getEffectiveLength